src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -357,13 +357,43 @@ private void Browser_Loaded(object sender, RoutedEventArgs e)
             LogToDynamoConsole(msg);
         }
 
+        /// <summary>
+        /// Collect the main and modifier key from KeyEventArgs in order to pass
+        /// that data to eventDispatcher (located in library.html) which is responsible
+        /// for binding KeyDown events between dynamo and webview instances
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        ///
+
+        // This enum is for matching the modifier keys between C# and javaScript
+        enum ModifiersJS
+        {
+            none = 0,
+            altKey = 1,
+            ctrlKey = 2,
+            shiftKey = 4
+        }
+
+        // This enum is for define the events to be tracked
+        enum EventsTracked
+        {
+            Delete,
+            C
+        }
+
         private void Browser_KeyDown(object sender, KeyEventArgs e)
 
         {
-            if (e.Key == Key.Delete)
+            if (!Enum.IsDefined(typeof(EventsTracked), e.Key.ToString())) return;
```

Comment:
The ToString() is not needed due that Enum.GetName returns a string

Reply:
Also if you type "C" this validation will be passed (not return), is that the expected behavior?

Reply:
> Also if you type "C" this validation will be passed (not return), is that the expected behavior?

There must be a modifier key involve, just pressing "C" key is not gonna trigger the Browser_KeyDown.

Reply:
> The ToString() is not needed due that Enum.GetName returns a string

I've decided to use .ToString() here in order to make it easier to follow as the Enum.isDefined syntax is already too large